### ISSUE TITLE
Trail issue

### ISSUE THREAD

**Issue by nolleto1** [LongIslandCW/morsebrowser#85]
The last card does not show up when using "Trail".  It stops playing and rewinds but the characters in the last card never appear.   Maybe need a short delay a the end before rewinding?

**Comment by RandyLoeb** [comment]
@nolleto1 yes it annoys me too. Based on the ICR discussion, I am thinking to add an adjustable delay before reveal, and like you suggest a last-word delay before doing the full rewind.

**Comment by RandyLoeb** [comment]
@nolleto1 @LongIslandCW  added some enhancements to trail. pre- delay is delay in seconds before revealing the card. post-delay is a delay in seconds after revealing the card. final is the delay after the final word, which I have set default to 1 second.

maybe only final is what we were after, but the other two delay settings allow customization and for trail to work like icr does with voice. another way to think about the pre-delay is "thinking time" and post-delay is "reading time" before the next morse hits.

take a look and let me know.

**Comment by nolleto1** [comment]
Randy,
I'm driving to Rolla MO tomorrow so won't be able to look at this until
tomorrow evening.  Will get back to you as soon as possible.
Tom

**Comment by nolleto1** [comment]
Randy,
Looks fine to me.  May be more than we need but it gives us a lot of flexibility.
Tom
